src/Editor.jsx

```diff
@@ -186,13 +206,13 @@ class Editor extends Component {
                         step: 2
                     });
 
-                    response.json().then(function(graph) {
+                    response.json().then(function (graph) {
```

Comment:
A few areas you're changing the function syntax here - the convention we use is:
```
function(a) {
    // ...
}
```

With no space between `function` and the open paren, so these don't need to change.

Reply:
Didn't catch that, sorry!